packages/synapse-react-client/src/components/FullTextSearch.tsx

```diff
@@ -36,23 +42,50 @@ export const FullTextSearch: React.FunctionComponent<FullTextSearchProps> = ({
     } else {
       executeQueryRequest(request => {
         const { additionalFilters = [] } = request.query
-
-        const textMatchesQueryFilter: TextMatchesQueryFilter = {
-          concreteType:
-            'org.sagebionetworks.repo.model.table.TextMatchesQueryFilter',
-          searchExpression: searchText,
-        }
-        // PORTALS-2093: does this additional filter already exist?
-        const found = additionalFilters.find(
-          filter =>
-            filter.concreteType == textMatchesQueryFilter.concreteType &&
-            filter.searchExpression == textMatchesQueryFilter.searchExpression,
-        )
-        if (found) {
-          return request
+        const idColumnModel = getColumnModel('id')
```

Comment:
Maybe use `getFileColumnModelId` (in SynapseTableUtils), unless that doesn't work for some known cases.

Reply:
Done